app/domain/authentication/authenticator_input.rb

```diff
@@ -24,17 +25,17 @@ def update(hash)
 
     def webservice
       @webservice ||= ::Authentication::Webservice.new(
-        account:            @account,
-        authenticator_name: @authenticator_name,
-        service_id:         @service_id
+        account:            account,
+        authenticator_name: authenticator_name,
+        service_id:         service_id
       )
     end
 
     # :reek:NilCheck
     def role
-      return nil if @username.nil?
+      return nil if username.nil?
 
-      ::Role.by_login(@username, account: @account)
+      ::Role.by_login(username, account: account)
```

Comment:
why did this change?

Reply:
API changes. The class variables in this current implementation are nill.